Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -187,7 +183,7 @@ This tutorial assumes a project using Swift Package Manager (SPM) for package de
 </p></details>
```

Comment:
Can you mark the "Discussion" section above as important and clarify the text:
Important: After this step, in this tutorial app there are no other Adobe Experience Cloud Solution extensions. At this point the import and registration for AEPIdentity extension can also be removed since AEPEdgeIdentity takes care of the identity piece on the Edge extensions. 
If your application still uses Adobe Experience Cloud Solution extensions, such as Adobe Target, Adobe Campaign (find the full list here -link- https://aep-sdks.gitbook.io/docs/), you should ignore this step and continue to use the AEPIdentity extension.

Reply:
I think warning is better for this one